app/README.md

```diff
@@ -0,0 +1,44 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Do we want to keep this README? It could be confusing to have two

Reply:
If we remove this readme then we might want to mention that the project was set up using CRA in the root readme

Reply:
Moved CRA docs to the root level readme and removed the CRA readme.